### DISCUSSION TITLE
Back Button

### DISCUSSION THREAD

**Discussion by Justbob56** [music-assistant/support#2843] [Feature requests and ideas]
It would be nice to have a "back" button in music assistant that can be turned on or off. When in Kiosk Mode or when using Fully Kiosk I can navigate to Music Assistant Server Interface inside Home Assistant but there is no back button to exit out to get back to the default dashboard.

**Comment by OzGav** [comment]
Please search before posting https://github.com/orgs/music-assistant/discussions/2736
